server.py

```diff
@@ -71,9 +71,7 @@ def do_POST(self):
 
 
 if __name__ == "__main__":
-  if not os.path.isfile("index.html"):
-    print("error: please launch this program in the same directory as index.html", file=sys.stderr)
-    sys.exit(1)
```

Comment:
I wouldn't get rid of this block. You try to chdir and then still do this check. This way if the files got moved the error can still show up.

Reply:
Good point! Fixed.